assets/js/payment-methods-demo/express-payment/apple-pay/apple-pay-express.js

```diff
@@ -0,0 +1,380 @@
+/**
+ * Internal dependencies
+ */
+import {
+	DEFAULT_STRIPE_EVENT_HANDLERS,
+	PAYMENT_METHOD_NAME,
+} from './constants';
+import {
+	getBillingData,
+	getPaymentMethodData,
+	normalizeShippingAddressForCheckout,
+	normalizeShippingOptions,
+	normalizeLineItems,
+} from './normalize';
+import {
+	getStripeServerData,
+	getPaymentRequest,
+	updatePaymentRequest,
+	canDoApplePay,
+	getTotalPaymentItem,
+	stripePromise,
+} from '../../stripe-utils';
+
+/**
+ * External dependencies
```

Comment:
I've noticed in recent files you've worked on you've started defining internal before external - is there a reason for this?

Reply:
Ha! I always get mixed up and go with whatever I remember doing last. It didn't even occur to me I was doing it wrong. If the convention is external first (which makes sense) I should definitely be consistent with that. I created an issue (#2052)